recipe/conda_build_config.yaml

```diff
@@ -411,9 +411,7 @@ giflib:
 glew:
   - 2.1
 glib:
-  - 2.58  # [not (osx and arm64)]
-  # Can't update to 2.66 on `osx and x86_64` yet because that requires a newer macOS version than 10.9
-  - 2.66  # [osx and arm64]
+  - '2.66'
```

Comment:
There was talk of changing this to `2` after the migration was done?

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1027#issuecomment-748063425

Reply:
@isuruf WDYT?

Reply:
`2` is fine with me.

Reply:
Yes, `2` is fine